crates/nu-command/src/filesystem/cp.rs

```diff
@@ -57,9 +57,9 @@ impl Command for Cp {
             // .switch("force", "suppress error when no file", Some('f'))
             .switch("interactive", "ask user to confirm action", Some('i'))
             .switch(
-                "no-dereference",
+                "no-symlink",
                 "If the -r option is specified, no symbolic links are followed.",
```

Comment:
The -r flag here should be fixed as well.

Reply:
The main purpose is that it's only useful when -r is active, I changed the description